Game/Scripts/Controller_FPSController.lua

```diff
@@ -185,4 +195,14 @@ function Controller_FPSController:CheckGround()
     end
 end
 
+function Controller_FPSController:UpdateCameraFOV(deltaTime)
+    local targetFOV = self.running and self.runAsked and self.initialFOV * self.runningFOVMultiplier or self.initialFOV
+
+    self.currentCameraFOV = Math.Lerp(self.currentCameraFOV, targetFOV, self.LerpSpeedFOV * deltaTime)
```

Comment:
We don't need to store the currentCameraFOV in the class/table, it can just be a local variable